routes/index.js

```diff
@@ -64,7 +64,7 @@ module.exports = (keystone) => {
 				return next();
 			}
 
-			res.render('job', job);
+			res.redirect(job.url_base);
```

Comment:
Not sure I follow what's supposed to happen here. In the current doc this field is undefined and you just get an error message?